configuration/pih/pih-config-mirebalais.json

```diff
@@ -70,6 +70,17 @@
       "mothersName": {
         "required": true
       }
+    },
+    "contactInfo": {
+      "phoneNumber": {
+        "regex": "(^\\d{8}$)|(^\\d{4}(?:[-| ])?\\d{4}$)|(^\\+?(?:\\d ?){7,14}\\d$)"
```

Comment:
The phone number format would be a change for EVERY ZL EMR – including HUM registration.  If you want to do this, I recommend that you notify HUM registration clerks ( can help).  I recommend you provide format tip (ie. xxxx-xxxx) with the label.

Otherwise when we do the next software upgrade at HUM, it will be difficult for users who do not use that format.

Today they can try anything into the telephone number:

Reply:
So I'd better do the two changes in different branches so that I could create a different pull request for the phone number validation?